### ISSUE TITLE
No hcloud?

### ISSUE THREAD

**Issue by jarlatsimplesite** [cbeneke/hcloud-fip-controller#28]
The documentation starts explaining requirements like servers etc, and I do have CX server in hetzner cloud, but no hcloud!?

**Comment by cbeneke** [comment]
Hi,

I don't really understand your question. `hcloud` is the CLI tool to manage hetzner cloud machines. This controller requires a running kubernetes cluster based on hetzner cloud machines, the hcloud CLI is not directly required, but can be used to e.g. create the hetzner cloud API token, which the controller needs to talk to the hetzner cloud API.

Could you rephrase your question for me? :)

**Comment by jarlatsimplesite** [comment]
Since I read the your document and currently know not too much about this, I might consider things being a problem without it being.
It's just when I call hcloud on my hetzner based server I'm told that the command is not found and I cannot apt install it. Thought that it might be in the hezter package repository..

**Comment by cbeneke** [comment]
This controller is bundled in a docker container, which includes the required bindings for the hcloud API. The cli tool itself is not required. Also please have in mind, that this controller is for Kubernetes only. Sorry for the docs not being really useful yet, hope to find the time in the near future to actually write some proper documentation.

If you want to run the hcloud binary, please refer to https://github.com/hetznercloud/cli on how to install and use it.

**Comment by cbeneke** [comment]
Closing here, since its not related to the repo
